lib/page/performMouseAction.js

```diff
@@ -30,6 +30,20 @@ const performMouseAction = async function performMouseAction(context, actionType
     return ({ status: STATUS.INVISIBLE });
   }
 
+  const overlappingNode = await context.page.evaluate(selectorNode => {
+    selectorNode.scrollIntoView({ block: 'center' });
+
+    const { left, top } = selectorNode.getBoundingClientRect();
+    const x = left + (selectorNode.clientWidth / 2);
+    const y = top + (selectorNode.clientHeight / 2);
+
+    const nodeEl = document.elementFromPoint(x, y);
+
+    return nodeEl && !selectorNode.contains(nodeEl) && nodeEl.className;
```

Comment:
If the function returns a classname, then we variable holding the result should show it somehow. Now it looks like `overlappingNode` contains an instance of Node.